src/daos/order_dao.py

```diff
@@ -8,7 +8,7 @@
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 
-FIELD_NAMES = ['market_slug', 'asset_id', 'price', 'size', 'timestamp']
+FIELD_NAMES = ['asset_id', 'market_slug', 'order_type', 'price', 'size', 'timestamp']
```

Comment:
We don't need to rearrange the 'market_slug' and 'asset_id'. Rerrange them in `Order` class instead to match this order. Also, let's add in `market_id`:

```suggestion
FIELD_NAMES = ['market_slug', 'market_id', 'asset_id', 'order_type', 'price', 'size', 'timestamp']
```

Update any tests necessary and the script and revert the data file updates before re-running the script.